lib/b2b/passwords_email.ts

```diff
@@ -0,0 +1,217 @@
+// !!!
+
+import { fetchConfig } from "../shared";
+import { Member, Organization } from "./organizations";
+import { MemberSession } from "./sessions";
+import { request } from "../shared";
+
+// Request type for `Email.reset`.
+export interface B2BEmailResetRequest {
+  // The password reset token to authenticate.
+  password_reset_token: string;
+  // The password to reset.
+  password: string;
+  /**
+   * Reuse an existing session instead of creating a new one. If you provide a `session_token`, Stytch will
+   * update the session.
+   *       If the `session_token` and `magic_links_token` belong to different Members, the `session_token`
+   * will be ignored. This endpoint will error if
+   *       both `session_token` and `session_jwt` are provided.
+   */
+  session_token?: string;
+  /**
+   * Set the session lifetime to be this many minutes from now. This will start a new session if one doesn't
+   * already exist,
+   *   returning both an opaque `session_token` and `session_jwt` for this session. Remember that the
+   * `session_jwt` will have a fixed lifetime of
+   *   five minutes regardless of the underlying session duration, and will need to be refreshed over time.
+   *
+   *   This value must be a minimum of 5 and a maximum of 527040 minutes (366 days).
+   *
+   *   If a `session_token` or `session_jwt` is provided then a successful authentication will continue to
+   * extend the session this many minutes.
+   *
+   *   If the `session_duration_minutes` parameter is not specified, a Stytch session will be created with a
+   * 60 minute duration. If you don't want
+   *   to use the Stytch session product, you can ignore the session fields in the response.
+   */
+  session_duration_minutes?: number;
+  /**
+   * Reuse an existing session instead of creating a new one. If you provide a `session_jwt`, Stytch will
+   * update the session. If the `session_jwt`
+   *       and `magic_links_token` belong to different Members, the `session_jwt` will be ignored. This
+   * endpoint will error if both `session_token` and `session_jwt`
+   *       are provided.
+   */
+  session_jwt?: string;
+  // A base64url encoded one time secret used to validate that the request starts and ends on the same device.
+  code_verifier?: string;
+  /**
+   * Add a custom claims map to the Session being authenticated. Claims are only created if a Session is
+   * initialized by providing a value in
+   *   `session_duration_minutes`. Claims will be included on the Session object and in the JWT. To update a
+   * key in an existing Session, supply a new value. To
+   *   delete a key, supply a null value. Custom claims made with reserved claims (`iss`, `sub`, `aud`,
+   * `exp`, `nbf`, `iat`, `jti`) will be ignored.
+   *   Total custom claims size cannot exceed four kilobytes.
+   */
+  session_custom_claims?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
+  locale?: "en" | "es" | "pt-br";
```

Comment:
Missing docs? I'll check upstream

Reply:
Also - is it possible to make this 
```
"en" | "es" | "pt-br" | string;
```

That way we can introduce new locales over time without needing to make everyone bump their library (though you won't get autocomplete / lang server help for new locales)

Reply:
This is part of the MFA changes so there aren't docs yet - coming soon!